ClientSideKeyVaultKeyToCustomerProvidedKey/config.py

```diff
@@ -0,0 +1,31 @@
+from azure.storage.blob import CustomerProvidedEncryptionKey
+from azure.keyvault.keys.crypto import KeyWrapAlgorithm
+
+# the below values are fake and need to be replaced
+
+KEYVAULT_URL = ""
+CLIENT_ID = ""
+CLIENT_SECRET = ""
+TENANT_ID = ""
+connection_str = ""
+
+# items to make encryption scope
+SUB_ID = ""
+RESOURCE_GROUP = ""
+STORAGE_ACCOUNT = ""
+
+# if using customer provided key, set values here
+customer_key = CustomerProvidedEncryptionKey(key_value="",
+                                             key_hash="")
+
+# keyvault key used for encryption
+keyname = ""
+secret = ""
+key_wrap_algorithm = KeyWrapAlgorithm.rsa_oaep #.rsa_oaep, .rsa_oaep_256, .rsa1_5, .aes_256
+
+cont_name = "client-side-keyvault-key-to-customer-provided-key"
+blob_name = "blobExample.txt"
+migrated_blob_name = "cpk-" + blob_name
+
+# if user wants overwrite to be available, change to True
```

Comment:
probably let's make it to:
# if user wants overwrite the existing blob which has the same name as `migrated_blob_name` , change to True
overwrite_existing_blob = False

What's your opinion?

Reply:
Yeah I'm thinking I will change it to "# if user wants to overwrite blob with the same name as migrated_blob_name, change value of overwriter to True"

Reply:
or technically it is also being used for the sampleDataCreator, so I think i will also include that if they want to overwrite the migrated_blob_name as well as the blob uploaded in the sampleDataCreator